openshift-origin-rhel/azuredeploy.json

```diff
@@ -2,19 +2,27 @@
 	"$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
 	"contentVersion": "1.0.0.0",
 	"parameters": {
+		"baseUrl": {
```

Comment:
please see the best guidance on _artifactsLocation and _artifactsLocationSasToken: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md